portfolio/src/main/webapp/index.html

```diff
@@ -15,6 +15,8 @@ <h1>Maryam Sharif's Portfolio</h1>
       <h3> GO BLUE! </h3> 
       <p> Click here for a random fact about me: <button onclick="addRandomGreeting()">Hello</button></p>
       <div id="greeting-container"></div>
+      <p>Click here to see a random message: <button onclick="getMessage()">Fetch a message!</button></p>
+      <div id="dataservlet-container"></div>
```

Comment:
You can rename this as something more meaningful, for example: messages-container